### PR TITLE
CSS in JS description update

### PR DESCRIPTION
**Pull request by georgewrmarshall**
Closes: #103

Updates descriptions of CSS in JS object for colors

### CODE REVIEW

File: src/colors/index.ts [hunk 1]
```diff
@@ -76,59 +76,59 @@ export interface Colors {
...
- * @property {string} background.default - Should be used as the default background color for any neutral type components
- * @property {string} background.alternative - Should be used as an alternative background for any neutral type components that require some slight contrast to background.default
+ * @property {string} background.default - For default neutral backgrounds
+ * @property {string} background.alternative - For a subtle contrast option for neutral backgrounds. (Example: backdrop, header background)
  *
- * @property {string} text.default - Should be used for all general text that takes main priority in the information hierarchy
- * @property {string} text.alternative - Should be used for all general text that takes less priority in the information hierarchy than text.default
- * @property {string} text.muted - Should be used for all low priority text: placeholder and inactive text. It will also not meet AAA or AA accessibility standards
+ * @property {string} text.default - For general text that takes main priority in the information hierarchy
+ * @property {string} text.alternative - For a weaker contrast option for neutral tex
```
Comment (Akatori-Design) [inline src/colors/index.ts:83]: noticing u missed a "t" at the end. maybe just in JS. looks fine in figma token

**Review by Akatori-Design — APPROVED** [review]

File: src/colors/index.ts (hunk 1 above)
Comment (georgewrmarshall) [reply src/colors/index.ts]: Udpated

**Review by Akatori-Design — APPROVED** [review]
🎯

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**